README.md

```diff
@@ -0,0 +1,101 @@
+#License MPL 2.0
+
+
```

Comment:
Please remove above this line (we linked MPL 2.0 licence from the top page), thanks!